### ISSUE TITLE
TeachingPopover

### ISSUE THREAD

**Issue by brunnogrb** [grbtec/gbt-fluent2-web#36]
- [ ] mountNode
- [ ] appearance
- [ ] children
- [ ] closeOnScroll
- [ ] defaultOpen
- [ ] inline
- [ ] mouseLeaveDelay
- [ ] withArrow
- [ ] onOpenChange
- [ ] open
- [ ] openOnContext
- [ ] openOnHover
- [ ] closeOnIframeFocus
- [ ] positioning
- [ ] size

- [ ] trapFocus

legacyTrapFocus I Must be used with the trapFocus prop Enables older Fluent UI focus trap behavior where the user cannot tab into the window outside of the document. This is now non-standard behavior according to the [HTML dialog spec](https://developer.mozilla.org/en-US/docs/Web/API/HTMLDialogElement/showModal) where the focus trap involves setting outside elements inert.

- [ ] inertTrapfocus
 IEnables standard behavior according to the [HTML dialog spec](https://developer.mozilla.org/en-US/docs/Web/API/HTMLDialogElement/showModal) where the focus trap involves setting outside elements inert, making navigation leak from the trapped area back to the browser toolbar and vice-versa.

- [ ] unstable_disableAutoFocus
